app/forms/nsm/youth_court_fee_form.rb

```diff
@@ -0,0 +1,48 @@
+module Nsm
+  class YouthCourtFeeForm < BaseAdjustmentForm
+    COMMENT_FIELD = 'youth_court_fee_adjustment_comment'.freeze
+    LINKED_CLASS = V1::AdditionalFee
+
+    attribute :remove_youth_court_fee
+    attribute :youth_court_fee_adjustment_comment
+
+    def save
+      return false unless valid?
+
+      remove_bool = ActiveModel::Type::Boolean.new.cast(remove_youth_court_fee)
```

Comment:
Could you not do `attribute :remove_youth_court_fee, :boolean` on line 6 and save yourself having to cast it manually here and do string comparisons on line 31?